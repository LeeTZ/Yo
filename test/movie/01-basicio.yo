# This script converts a movie to any other format. Details could be configured in config.ini
a = Clip("mactest.mov")
a.save("mactest.webm")
