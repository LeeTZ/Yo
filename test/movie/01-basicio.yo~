a = Clip("mactest.mov")
a.save("mactest.webm")