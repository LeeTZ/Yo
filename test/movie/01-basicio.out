{
   "alpha" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 0.0
            },
            "interpolation" : 2
         }
      ]
   },
   "anchor" : 0,
   "crop_height" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "crop_width" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "crop_x" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 0.0
            },
            "interpolation" : 2
         }
      ]
   },
   "crop_y" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 0.0
            },
            "interpolation" : 2
         }
      ]
   },
   "duration" : 8.893242835998535,
   "effects" : [],
   "end" : 8.893242835998535,
   "gravity" : 4,
   "id" : "",
   "layer" : 0,
   "location_x" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 0.0
            },
            "interpolation" : 2
         }
      ]
   },
   "location_y" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 0.0
            },
            "interpolation" : 2
         }
      ]
   },
   "perspective_c1_x" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "perspective_c1_y" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "perspective_c2_x" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "perspective_c2_y" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "perspective_c3_x" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "perspective_c3_y" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "perspective_c4_x" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "perspective_c4_y" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : -1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "position" : 0.0,
   "reader" : {
      "acodec" : "aac",
      "audio_bit_rate" : 63669,
      "audio_stream_index" : 1,
      "audio_timebase" : {
         "den" : 44100,
         "num" : 1
      },
      "channel_layout" : 3,
      "channels" : 2,
      "display_ratio" : {
         "den" : 2,
         "num" : 3
      },
      "duration" : 8.893242835998535,
      "file_size" : "3682217",
      "fps" : {
         "den" : 1,
         "num" : 24
      },
      "has_audio" : true,
      "has_single_image" : false,
      "has_video" : true,
      "height" : 720,
      "interlaced_frame" : false,
      "pixel_format" : 0,
      "pixel_ratio" : {
         "den" : 1,
         "num" : 1
      },
      "sample_rate" : 44100,
      "top_field_first" : true,
      "type" : "FrameMapper",
      "vcodec" : "h264",
      "video_bit_rate" : 3289466,
      "video_length" : "213",
      "video_stream_index" : 0,
      "video_timebase" : {
         "den" : 24,
         "num" : 1
      },
      "width" : 1080
   },
   "rotation" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 0.0
            },
            "interpolation" : 2
         }
      ]
   },
   "scale" : 1,
   "scale_x" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "scale_y" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "shear_x" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 0.0
            },
            "interpolation" : 2
         }
      ]
   },
   "shear_y" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 0.0
            },
            "interpolation" : 2
         }
      ]
   },
   "start" : 0.0,
   "time" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 0.0
            },
            "interpolation" : 2
         }
      ]
   },
   "volume" : {
      "Points" : [
         {
            "co" : {
               "X" : 1.0,
               "Y" : 1.0
            },
            "interpolation" : 2
         }
      ]
   },
   "wave_color" : {
      "alpha" : {
         "Points" : [
            {
               "co" : {
                  "X" : 1.0,
                  "Y" : 255.0
               },
               "handle_left" : {
                  "X" : 1.0,
                  "Y" : 255.0
               },
               "handle_right" : {
                  "X" : 1.0,
                  "Y" : 255.0
               },
               "handle_type" : 0,
               "interpolation" : 0
            }
         ]
      },
      "blue" : {
         "Points" : [
            {
               "co" : {
                  "X" : 1.0,
                  "Y" : 255.0
               },
               "handle_left" : {
                  "X" : 1.0,
                  "Y" : 255.0
               },
               "handle_right" : {
                  "X" : 1.0,
                  "Y" : 255.0
               },
               "handle_type" : 0,
               "interpolation" : 0
            }
         ]
      },
      "green" : {
         "Points" : [
            {
               "co" : {
                  "X" : 1.0,
                  "Y" : 123.0
               },
               "handle_left" : {
                  "X" : 1.0,
                  "Y" : 123.0
               },
               "handle_right" : {
                  "X" : 1.0,
                  "Y" : 123.0
               },
               "handle_type" : 0,
               "interpolation" : 0
            }
         ]
      },
      "red" : {
         "Points" : [
            {
               "co" : {
                  "X" : 1.0,
                  "Y" : 0.0
               },
               "handle_left" : {
                  "X" : 1.0,
                  "Y" : 0.0
               },
               "handle_right" : {
                  "X" : 1.0,
                  "Y" : 0.0
               },
               "handle_type" : 0,
               "interpolation" : 0
            }
         ]
      }
   },
   "waveform" : false
}
